package.json

```diff
@@ -130,16 +130,15 @@
   },
   "devDependencies": {
     "@storybook/addon-actions": "^6.1.21",
-    "@storybook/addon-essentials": "^6.1.21",
+    "@storybook/addon-essentials": "^6.2.9",
     "@storybook/addon-links": "^6.1.21",
     "@storybook/node-logger": "^6.1.21",
     "@storybook/preset-create-react-app": "^3.1.7",
-    "@storybook/react": "^6.1.21",
+    "@storybook/react": "^6.2.9",
```

Comment:
How did this update? We should try and avoid updating packages unrelated to the task (and instead perhaps update these packages related to `storybook` together)

Reply:
sorry, that's totally my fault! should I change it back and commit again?

Reply:
Yeah just revert those two bumps and push a new commit :+1: You can likely do:
```
npm install @storybook/addon-essential@6.1.21 storybook/react@6.1.21
```